packages/stores/src/account/osmosis/index.ts

```diff
@@ -21,10 +21,14 @@ import { DeliverTxResponse } from "../types";
 import { findNewClPositionId } from "./tx-response";
 import { DEFAULT_SLIPPAGE, osmosisMsgOpts } from "./types";
 
+interface CoinType {
+  currency: Currency;
+  amount: string;
+  denom: Currency;
+}
```

Comment:
Fe things here:

* I'd prefer this bespoke type be inlined, as it makes the parameter type of the function easier to see immediately with the IDE.
* You repeat the Currency twice, if the denom should be a string, that would also repeat the denom info found in the first Currency member

Reply:
I've refactored to remove `currency` but the reason I extracted into an interface is because this is used multiple times.  This keeps it DRYer

Reply:
DRY is nice but not a hard and fast rule. I prefer readability, proximity, explicitness, etc. over DRYness in situations like this